app/main.py

```diff
@@ -1,8 +1,27 @@
 class Person:
-    # write your code here
-    pass
+    people = dict()
 
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.people[name] = self
 
-def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+
+def create_person_list(people: list) -> list[Person]:
+    def get_instance(name: str) -> Person:
+        for person in person_instances:
+            if person.name == name:
+                return person
+
+    person_instances = [
+        Person(person["name"], person["age"])
+        for person in people
+    ]
```

Comment:
This can fit on one line

Reply:
flake8 said that line was too long so i fitted it on 2 lines